build_for_ultimaker.sh

```diff
@@ -77,7 +77,7 @@ run_build()
 
 deliver_pkg()
 {
-    cp "${BUILD_DIR}/"*".deb" "./"
+    cp "${BUILD_DIR}/"*".deb" "./" 2> /dev/null
```

Comment:
Why do you throw away stderr?

Reply:
Because for some targets (like, clean, menuconfig, oldconfig, etc) there will be no package created, so just ignores the absence of the package.